cypress/e2e/notes-100/access-shared-note.cy.js

```diff
@@ -1,16 +1,39 @@
 import {waitForModel, homepageSetup} from '../../support/utils'
 
-
-describe('select-a-note', () => {
-  context('Open the link with a shared note', () => {
+// From https://github.com/bldrs-ai/Share/issues/1072
+describe('Notes 100 - Access shared note', () => {
+  context('Access shared link containing the path to the general notes', () => {
     beforeEach(() => {
       homepageSetup()
-    })
-    it('Visit notes permalink, side drawer shall be opened', () => {
       cy.setCookie('isFirstTime', '1')
       cy.visit('/share/v/p/index.ifc#c:-133.022,131.828,161.85,-38.078,22.64,-2.314;i:')
       waitForModel()
+    })
+    it('Panel title to contain NOTES string', () => {
       cy.get('[data-testid="panelTitle"]').contains('NOTES')
     })
+    it('Notes list to be visible', () => {
+      cy.get('.MuiList-root').should('exist')
+    })
+  })
+  context('Access the shared link containing the path to a note index', () => {
+    beforeEach(() => {
+      homepageSetup()
```

Comment:
remove homepage and setcookie

Reply:
But we need it to setup for the context, no?

Reply:
The way you have it here is necessary, but the idea was to have a common beforeEach for homepage+returning use for both contexts

In general, you shouldn't have 2 beforeEach's in a row, like this